golang-github-osbuild-composer.spec

```diff
@@ -26,14 +26,15 @@ BuildRequires:  systemd
 BuildRequires:  systemd-rpm-macros
 BuildRequires:  git
 BuildRequires:  golang(github.com/aws/aws-sdk-go)
-BuildRequires:  golang-github-azure-storage-blob-devel
+BuildRequires:  golang(github.com/Azure/azure-storage-blob-go/azblob)
```

Comment:
What's actually going on here? This is different from the other `golang()` requires, no?

Reply:
These golang packages simple include a `Provide` that looks like this. That one package does it slightly differently. I think it's still worth the consistency. I've added an explanation to the commit message.